extensions/wikia/AdEngine/js/video/player/porvata/googleIma.js

```diff
@@ -23,7 +23,14 @@ define('ext.wikia.adEngine.video.player.porvata.googleIma', [
 
 	function getPlayer(params) {
 		var adDisplayContainer = new win.google.ima.AdDisplayContainer(params.container),
-			adsLoader = new win.google.ima.AdsLoader(adDisplayContainer);
+			adsLoader,
+			iframe = params.container.querySelector('div > iframe');
+
+		// TODO: remove this hack
+		// it's reloading iframe in order to make IMA work when user is moving back to the page with player
```

Comment:
You've mentioned an up-to-date thread about it which URL I suggest leaving in the comment ;) I found and issue from 2015: https://github.com/googleads/videojs-ima/issues/110 :/